schema.graphql

```diff
@@ -1,8 +1,95 @@
+type Account @entity {
+    accountId: ID!
+    address: ID!
+    daoAvatarAddress: ID!
+    reputation: BigInt
+    tokens: BigInt
+    stakes: [Stake!] @derivedFrom(field: "accountId")
+    votes: [Vote!] @derivedFrom(field: "accountId")
+    proposals: [Proposal!] @derivedFrom(field: "proposer")
+    redemptions: [Redemption] @derivedFrom(field: "accountId")
+}
+
+type DAO @entity {
+    avatarAddress: ID!
+    controllerAddress: Bytes
+    members: [Account!] @derivedFrom(field: "daoAvatarAddress")
```

Comment:
daoAvatarAddress should be avatarAddress ?

Reply:
All addresses should be Address type (instead of Bytes)
https://github.com/graphprotocol/graph-ts/blob/25591ce9384ce9428e7b789b551e182ba4587596/index.ts#L144

Reply:
> All addresses should be Address type (instead of Bytes)
> https://github.com/graphprotocol/graph-ts/blob/25591ce9384ce9428e7b789b551e182ba4587596/index.ts#L144

I agree and I'm updating it to Address instead of bytes. But keeping it as ID wherever it is unique identifier, since it is serialized in same way just makes it explicit that the field is a unique identifier

Reply:
Oops. Address in not a valid type for graphql schema. Its an ethereum type.